awx/api/generics.py

```diff
@@ -53,6 +54,29 @@
 analytics_logger = logging.getLogger('awx.analytics.performance')
 
 
+class LoggedLoginView(auth_views.LoginView):
+
+    def post(self, request, *args, **kwargs):
```

Comment:
A similar comment that I was making - it appears that you can perform a GET in order to receive the HTML login form, but since `get` is not defined in this view, it can't deliver any kind of JSON response (or error code). I think that resulted in curl commands getting the HTML. Anyway, this should get some more testing and polish.